lighthouse-core/config/perf-config.js

```diff
@@ -6,7 +6,7 @@
 'use strict';
 
 module.exports = {
-  extends: 'lighthouse:default',
+  extends: 'lighthouse:observed',
```

Comment:
I feel like the "observed" name refers to observed things (metrics, etc), regardless of throttling method.

Since this config is primarily about changing throttling method I would have expected a name like 'throttled'

Reply:
I started out with `devtools` as the name but the primary value of this thing existing is having the 5s+ wait times in the defaultPass which applies to all observed methods. It does indeed change the throttlingMethod as well, but that's easily done without presets/extension. `throttled` overly communicates the value in changing the throttling IMO, i.e. why would I want `--preset=throttled --throttlingMethod=off/provided`?

The most consistent might be to make `observed` not actually apply the throttling by default, and require `--throttlingMethod=devtools` but that seems to discourage throttling which isn't great.

Reply:
I just think its confusing that observedMetrics all exist outside of the `lighthouse:observed` config.

<!--
how about..

> precise
> slow
> strict
> original
-->

better question: when is observed used? for --preset=perf and for the ttci smoketest, right? why doesn't ttci smoke just use/extend perf config?

Reply:
agree that `observed` is a little confusing here. It does sound like it's fairly neutral, observed regardless of throttling method, but if I was told it affected the throttling method I would guess it was closer to `none`, not using devtools throttling.

But it sounds like you may be rearranging things, so maybe I'll wait to comment further :)